ios/PosthogReactNativeSessionReplay.swift

```diff
@@ -26,6 +26,12 @@ class PosthogReactNativeSessionReplay: NSObject {
     let maskAllImages = sdkReplayConfig["maskAllImages"] as? Bool ?? false
     config.sessionReplayConfig.maskAllImages = maskAllImages
 
+    let maskAllSandboxedViews = sdkReplayConfig["maskAllSandboxedViews"] as? Bool ?? false
+    config.sessionReplayConfig.maskAllSandboxedViews = maskAllSandboxedViews
+
+    let maskPhotoLibraryImages = sdkReplayConfig["maskPhotoLibraryImages"] as? Bool ?? false
+    config.sessionReplayConfig.maskPhotoLibraryImages = maskPhotoLibraryImages
```

Comment:
```suggestion
    let maskAllSandboxedViews = sdkReplayConfig["maskAllSandboxedViews"] as? Bool ?? true
    config.sessionReplayConfig.maskAllSandboxedViews = maskAllSandboxedViews

    let maskPhotoLibraryImages = sdkReplayConfig["maskPhotoLibraryImages"] as? Bool ?? true
    config.sessionReplayConfig.maskPhotoLibraryImages = maskPhotoLibraryImages
```

lets keep the default value

Reply:
I think maskAllTextInputs, maskAllImages and captureNetworkTelemetry should always fallback to `true` since its the default behaviour, mind changing it?

Reply:
> I thought about this when was implementing the change and in this case this means that native code was called with the wrong options somehow

its possible if people use JS and not TS without the typechecks but that's ok, the mistake would be on their end.
Always use the default value as a fallback, its the safest (disabling means not masking - PII risk etc)
if-let is also ok but I'd just keep with the default value since its less things to change/test again.